### ISSUE TITLE
Problem with destroy

### ISSUE THREAD

**Issue by podviaznikov** [imgix/luminous#22]
I create few `Luminous` object dynamically and then trying to `destroy` them in the react application on `componentWillUnmount`.

It seems like destroy will throw an error if Lightbox was never activated:

```javascript
value: function _unbindEvents() {
      this.trigger.removeEventListener(this.settings.openTrigger, this.open, false);
      this.lightbox.el.removeEventListener(this.settings.closeTrigger, this.close, false);

      if (this.settings.closeWithEscape) {
        window.removeEventListener('keyup', this._handleKeyup, false);
      }
```
in the code above `this.lightbox.el` can be `undefined` when `this.lighbox.elementBuilt ` is false

**Comment by paulstraw** [comment]
Good catch @podviaznikov! This is fixed and released as `0.2.7`: https://github.com/imgix/luminous/releases/tag/0.2.7

**Comment by podviaznikov** [comment]
Perfect, thank you for the fix!
